include/deal.II/multigrid/mg_constrained_dofs.h

```diff
@@ -87,9 +87,17 @@ class MGConstrainedDoFs : public Subscriptor
   /**
    * Return the indices of dofs for each level that are subject to boundary
    * constraints.
+   *
+   * @deprecated: use get_boundary_indices(const unsigned int) instead.
    */
   const std::vector<std::set<types::global_dof_index> > &
-  get_boundary_indices () const;
+  get_boundary_indices () const DEAL_II_DEPRECATED;
+
+  /**
+   * Return the indices of dofs for each level that are subject to boundary constraints.
+   */
+  const IndexSet &
+  get_boundary_indices (const unsigned int level) const;
```

Comment:
I suppose the documentation should state that the index set only contains locally owned (active? relevant?) dofs if the triangulation is parallel?